Codox fail when using lein managed dependencies

```
Exception in thread "main" java.lang.AssertionError: Assert failed: (dep-spec? dep-spec)
        at leinjacker.deps$dep_name.invokeStatic(deps.clj:23)
        at leinjacker.deps$dep_name.invoke(deps.clj:23)
        at leinjacker.deps$has_dep_QMARK_$fn__841.invoke(deps.clj:41)
        at clojure.core$some.invokeStatic(core.clj:2592)
        at clojure.core$some.invoke(core.clj:2583)
        at leinjacker.deps$has_dep_QMARK_.invokeStatic(deps.clj:41)
        at leinjacker.deps$has_dep_QMARK_.invoke(deps.clj:30)
        at leinjacker.deps$add_if_missing.invokeStatic(deps.clj:50)
        at leinjacker.deps$add_if_missing.invoke(deps.clj:43)
```
This exception is due to leinjacker which doesn't support managed dependencies. 
The last snapshot version has a fix for this issue:
https://github.com/sattvik/leinjacker/commit/3f4b0d73170dcd984608c703d80ee4d52dcdac73

--- a/lein-codox/project.clj
+++ b/lein-codox/project.clj
@@ -5,5 +5,5 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :eval-in-leiningen true
-  :dependencies [[leinjacker "0.4.2"]
+  :dependencies [[leinjacker "0.4.3"]
                  [org.clojure/core.unify "0.5.7"]])

